### ISSUE TITLE
Use sampled softmax in Keras

### ISSUE THREAD

**Issue by limapedro** [keras-team/keras#10681]
I'm training a seq2seq using keras, since my model has a lot of classes I'm looking for ways to speed up the training, I found the sampled softmax function, any clue on how to use it in Keras?

I've tried without sucsess, my current code in the compile function.

model.compile(optimizer='adam', loss=tf.nn.sampled_softmax_loss, metrics=['acc'])
